### PR TITLE
Documentation: Table of abbreviations

### PR DESCRIPTION
**Pull request by maawoo**
This adds a table of abbreviations to the documentation in the "about"-section.

### CODE REVIEW

File: docs/about/abbreviations.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+   * - SM
+     - Stripmap Mode
+   * - SNAP
+     - Sentinel Application Platform
+   * - STAC
+     - SpatioTemporal Asset Catalog
+   * - UTM
+     - Universal Transverse Mercator
+   * - VRT
+     - Virtual Raster Format
```
Comment (johntruckenbrodt) [inline docs/about/abbreviations.rst:79]: Virtual Raster Tile

**Review by johntruckenbrodt — CHANGES_REQUESTED** [review]
two minor changes..

File: docs/about/abbreviations.rst (hunk 1 above)
Comment (maawoo) [reply docs/about/abbreviations.rst:79]: Ah yeah, I'm blaming Copilot on this one... 😅

File: docs/about/abbreviations.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+   * - CRS
+     - Coordinate Reference System
+   * - DEM
+     - Digital Elevation Model
+   * - DSM
+     - Digital Surface Model
+   * - EGM
+     - Earth Gravitational Model
+   * - EPSG
+     - European Petroleum Survey Group
```
[the remark this answers is not in the document]
Comment (maawoo) [reply docs/about/abbreviations.rst]: Done https://github.com/SAR-ARD/S1_NRB/pull/123/commits/f2fa33c013f5b967fd86bcb0adca20f225cd8aaf

File: docs/about/abbreviations.rst (hunk 1 above)
Comment (maawoo) [reply docs/about/abbreviations.rst]: Done https://github.com/SAR-ARD/S1_NRB/pull/123/commits/f2fa33c013f5b967fd86bcb0adca20f225cd8aaf

**Review by johntruckenbrodt — APPROVED** [review]

**Comment by johntruckenbrodt** [comment]
thanks @maawoo

**Pull request merged**